src/client-apps/nuxeo-drive/nuxeo-drive-direct-edit.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Direct Edit
+description: 'Direct Edit enables you to edit any of your document content from their Summary tab even if they are not synchronized.'
```

Comment:
"Summary tab" was for JSF I guess, right? Now the icon is simply put on each documents, in the "actions menu".